tests/disruption_helpers.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.framework import config
+
+
+logger = logging.getLogger(__name__)
+
+POD = ocp.OCP(kind=constants.POD, namespace=config.ENV_DATA['cluster_namespace'])
+
+
+class Disruptions:
+    """
+    This class contains methods of disrupt operations
+    """
+    resource = None
+    resource_obj = None
+    resource_count = 0
+
+    def set_resource(self, resource):
+        self.resource = resource
+        if self.resource == 'mgr':
+            self.resource_obj = pod.get_mgr_pods()
+        if self.resource == 'mon':
+            self.resource_obj = pod.get_mon_pods()
+            self.resource_count = 3
+        if self.resource == 'osd':
+            self.resource_obj = pod.get_osd_pods()
+        if self.resource == 'mds':
+            self.resource_obj = pod.get_mds_pods()
+            self.resource_count = 2
+
+    def delete_resource(self):
+        self.resource_obj[0].delete(force=True)
```

Comment:
@ebenahar so now instead of deleting all the pods, we shall be deleting only one ? 
elf.resource_obj[0]

Reply:
yes, I've changed it to only one for now.
In case you'll find this flow irrelevant, feel free to change it. I'd like this PR to cover the basic flow

Reply:
No this is good for a basic tier1 - where we don't expect operations to be halted due to these delete operations.. 1 pod delete shouldn't affect any operation :)